nunit-project-editor.nuspec

```diff
@@ -0,0 +1,25 @@
+﻿<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>nunit-project-editor</id>
+    <version>0.0.0</version>
```

Comment:
Should this use `$version$` like our other nuspecs, to force the value to be passed as a parameter, or did that not work for chocolatey?